Formula/python@3.11.rb

```diff
@@ -331,6 +331,22 @@ def install
     }.each do |short_name, long_name|
       (libexec/"bin").install_symlink (bin/long_name).realpath => short_name
     end
+
+    # Mark Homebrew python as externally managed: https://peps.python.org/pep-0668/#marking-an-interpreter-as-using-an-external-package-manager
+    (HOMEBREW_PREFIX/"lib/python#{version.major_minor}/EXTERNALLY-MANAGED").write <<~EOS
```

Comment:
```suggestion
    (site_packages/"EXTERNALLY-MANAGED").write <<~EOS
```
Though I think you may actually need `site_packages_cellar` here? Not sure. @Bo98 might know more.

Reply:
Also if you're writing directly to `HOMEBREW_PREFIX` then I think this needs to be in `post_install`.

Reply:
Yeah, wasn't sure about the location here. I just ended up going with the closest equivalent of upstream docs, but maybe shipping it in the Cellar is better.

Reply:
`lib_cellar` looks correct here, based on the documentation that it searches `stdlib`.